orthos2/frontend/templates/frontend/base.html

```diff
@@ -23,7 +23,7 @@
             <span style="color: #fff; font-size: 30px; font: Gotham, Arial, sans-serif;"><strong>Orthos</strong></span>
           </a>
         </div>
-        {% include 'snippet.userbar.html' %}
+        {% include 'frontend/snippet.userbar.html' %}
```

Comment:
Not sure this is the first one.
We agreed to not move all the templates for now (ideally never).